Cython/Utility/ObjectHandling.c

```diff
@@ -1992,6 +1992,25 @@ typedef struct {
 /////////////// UnpackUnboundCMethod ///////////////
 //@requires: PyObjectGetAttrStr
 
+#define CONST_REINTERPRET_CAST(pointer_type, pointer) ((pointer_type)(void *)(pointer))
```

Comment:
I don't think function pointer->void* is actually portable. I believe function pointer-> other function pointer is allowed though.

Also, if we want to keep these they should probably have `__PYX_` names

Reply:
> > Also, if we want to keep these they should probably have __PYX_ names

> Sure! I can see some areas that can benifit from clean up with those macros, if I want to refactor them it should probably be done in another pr

For now I'm just suggesting changing changing the name to not clash with anything a user/library might have defined. I agree a refactor of elsewhere shouldn't be in this PR